zerver/tests/test_submessage.py

```diff
@@ -115,7 +115,8 @@ def test_endpoint_success(self) -> None:
             content='{"name": "alice", "salary": 20}',
         )
         with mock.patch("zerver.lib.actions.send_event") as m:
-            result = self.client_post("/json/submessage", payload)
+            with self.captureOnCommitCallbacks(execute=True):
+                result = self.client_post("/json/submessage", payload)
```

Comment:
Can we instead have a prep commit refactor this test to use `tornado_redirected_to_list`?  It'd be nice to contain `self.captureOnCommitCallbacks` calls to just the two place we use this.

Reply:
This was done in #18626.